fixtures/build.js

```diff
@@ -22,28 +20,32 @@ window.EXAMPLE_BUILD = {
       }
     },
     "strategy": {
-      "type": "STI",
-      "stiStrategy": {
-        "builderImage": "openshift/ruby-20-centos7:latest",
-        "image": "openshift/ruby-20-centos7:latest"
+      "type": "Source",
+      "sourceStrategy": {
+        "from": {
+          "kind": "DockerImage",
+          "name": "openshift/ruby-20-centos7:latest"
+        },
+        "incremental": true
       }
     },
     "output": {
       "to": {
-        "kind": "ImageStream",
-        "name": "origin-ruby-sample"
-      },
-      "dockerImageReference": "172.30.73.77:5000/test/origin-ruby-sample",
-      "imageTag": "test/origin-ruby-sample",
-      "registry": "172.30.73.77:5000"
-    }
+        "kind": "ImageStreamTag",
+        "name": "origin-ruby-sample:latest"
```

Comment:
@bparees same here... do we know all the formats to expect for kinds of `'ImageStreamImage', 'ImageStreamTag' or 'DockerImage'`?